docs/user_guide/hopsworks/jobs.rst

```diff
@@ -26,13 +26,13 @@ follow the steps below:
 * Step 2: Give a name for you job
 * Step 3: Select one of the available job types
 * Step 4: Select the executable file of your job that you have uploaded earlier in a Dataset
-* Step 5 (Optional): Set the main class
+* Step 5 (Optional): Set the main class and default arguments to run the job with
```

Comment:
Actually master branch is wrong here, it's a chance to fix it. `main class` is mandatory for Spark jobs, the back-end will try to infer it and will fill in the textbox. If it is empty, the user needs to fill it in. So we could have a step5 for main class and step6 (optional) for default arguments

Reply:
Alright will fix it